### DISCUSSION TITLE
Non-interactive charts in Multi-Window Plot

### DISCUSSION THREAD

**Discussion by apaaris** [pyvista/pyvista#3252] [Q&A]
Hi everyone, 
I noticed that in Multi-Window plots the key events stop working. 
At the moment I have callbacks for three different key presses, I plot everything and select the proper subplot first, but then when pressing the keys nothing happens. The exactly same callbacks works with single window plots, so I guess it's due to the Multi-Window environment... Is there that can be done to have some interaction through key presses?

Also I noticed that interactive 2D plots stop being interactive in multi window plot even when the proper subplot is selected.

Thanks in advance

**Comment by banesullivan** [comment]
This doesn't sound good... would you please share some minimal code to reproduce these two problems?

**Reply by apaaris** [reply]
Also I managed to find out that the first problem was an error on my side, I accidentally reinitialized pv.Plotter after having set the key press events...

The second one still persists, I will attach a code snippet

**Comment by apaaris** [comment]
```python3
import numpy as np
import pyvista as pv
from pyvista import examples

class Display:
    
    def __init__(self):
        shape = (4, 1)  # 4 by 1 grid
        row_weights =  [ 1, 1, 1, 1]
        self.plotter = pv.Plotter(shape=shape, row_weights=row_weights)
        self.test2()
        
    def test2(self):
            #Dummy
            rng = np.random.default_rng(1)
            x = rng.standard_normal(100)
            y = rng.standard_normal(100)
            chart = pv.Chart2D()
            chart.grid = True
            chart.background_color = "white"
            chart.scatter(x, y, size=10, style="o",color="cyan")
            
            
            for i in range(4):
                self.plotter.subplot(i,0)
                self.plotter.add_chart(chart)
                
            self.plotter.show()
            
a = Display()
            
```

**Reply by dcbr** [reply]
Thanks for reporting this. This is definitely a bug and I created a new issue (#3278) to further investigate it.
